### ISSUE TITLE
Where is ReLu used??

### ISSUE THREAD

**Issue by kannan500** [jiecaoyu/XNOR-Net-PyTorch#90]
In the base paper,flow is described as 
Batch Normalisation --> Binary Activation -->Convolution -->MaxPooling

So,where in this flow,ReLu is used??

Similiarly,are you doing binarization after Batch Norm in your code?

**Comment by yaduydk97** [comment]
@jiecaoyu I also have the same doubt...Hope you will answer this soon

**Comment by jiecaoyu** [comment]
@kannan500 @yaduydk97 I am applying ReLU after the ```Convolution```. It is mentioned that ```We can insert a non-binary activation(e.g.,ReLU) after binary convolution.``` in the original paper (page.9).

**Comment by kannan500** [comment]
Thank you for your quick reply
